.github/workflows/pr-labeler.yml

```
@@ -1,10 +1,7 @@
 name: Pull request labeler
 
 on:
-  pull_request:
-    types: [opened, synchronize]
-    branches:
-      - "main"
+  - pull_request_target
```

I think a hybrid approach from the commits works (I'm not sure we need the issues permission though):

```suggestion
  - pull_request_target
  
 permissions:
  pull-requests: write
  contents: read
```

An approach like this with `pull_request_target` is in the github docs for a similar situation to ours: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#example-1-passing-the-github_token-as-an-input.

And to lock things down even more, I think we want to add `persist-credentials: false` to the checkout step, to ensure the key doesn't get cached locally anywhere.